composer.json

```diff
@@ -7,63 +7,65 @@
         "drupal/description_list_field": "1.0.x-dev",
         "drupal/drupal-driver": "^2.1.1",
-        "drupal/drupal-extension": "~4.0",
+        "drupal/drupal-extension": "~4.1",
         "drupal/emr": "~1.0-beta9",
-        "drupal/entity_reference_revisions": "~1.3",
-        "drupal/entity_browser": "^2.5",
-        "drupal/extra_field": "^1.1",
-        "drupal/field_group": "~3.0",
+        "drupal/entity_reference_revisions": "~1.9",
+        "drupal/entity_browser": "^2.6",
+        "drupal/extra_field": "^1.3",
+        "drupal/field_group": "^3.1",
         "drupal/inline_entity_form": "~1.0-rc9",
-        "drupal/json_field": "^1.0.0-rc2",
+        "drupal/json_field": "^1.0.0-rc4",
         "drupal/renderviz": "^1.0@alpha",
-        "drupal/styleguide": "~1.0-alpha3",
+        "drupal/styleguide": "~1.0@beta",
         "drupal/typed_link": "~1.1",
-        "drush/drush": "~9.0",
+        "drush/drush": "~10.3",
         "easyrdf/easyrdf": "1.0.0 as 0.9.1",
         "friends-of-behat/service-container-extension": "^1.0",
-        "guzzlehttp/guzzle": "~6.3",
         "instaclick/php-webdriver": "^1.4.7",
         "nikic/php-parser": "~3.0",
         "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "~1.5",
-        "openeuropa/drupal-core-require-dev": "^8.9.14",
-        "openeuropa/oe_content": "2.x-dev",
-        "openeuropa/oe_corporate_blocks": "~4.0",
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9.1",
+        "openeuropa/oe_content": "dev-EWPP-911",
+        "openeuropa/oe_corporate_blocks": "~4.1",
         "openeuropa/oe_corporate_countries": "~2.0",
         "openeuropa/oe_media": "~1.12",
-        "openeuropa/oe_multilingual": "dev-master",
-        "openeuropa/oe_paragraphs": "dev-master",
-        "openeuropa/oe_search": "~1.0",
-        "openeuropa/oe_webtools": "~1.10",
-        "openeuropa/oe_contact_forms": "~1.0",
+        "openeuropa/oe_multilingual": "dev-OEL-301",
+        "openeuropa/oe_paragraphs": "dev-OEL-67",
+        "openeuropa/oe_search": "dev-OEL-60",
+        "openeuropa/oe_webtools": "~1.12",
+        "openeuropa/oe_contact_forms": "dev-OEL-164",
         "openeuropa/rdf_skos": "~1.0",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
-        "phpunit/phpunit": "~6.0",
-        "symfony/dom-crawler": "~3.4"
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0.0-beta3",
+        "phpspec/prophecy-phpunit": "^1 || ^2",
+        "symfony/dom-crawler": "~4.4"
     },
     "_readme": [
         "We explicitly require consolidation/robo to allow lower 'composer update --prefer-lowest' to complete successfully.",
-        "We explicitly require consolidation/annotated-command to allow lower 'composer update --prefer-lowest' to complete successfully."
+        "We explicitly require consolidation/annotated-command to allow lower 'composer update --prefer-lowest' to complete successfully.",
```

Comment:
Let's add an explanation for why we've used `^8.9.16` as constraint and not `^8.9`.